README.md

```diff
@@ -61,6 +61,8 @@ versions by running `aws s3 ls s3://amazon-eks/cloudformation/`.
 
 | CloudFormation Version | EKS AMI versions                           | [amazon-vpc-cni-k8s](https://github.com/aws/amazon-vpc-cni-k8s/releases) |
 | ---------------------- | ------------------------------------------ | -------------------- |
+| 2019-09-27             | amazon-eks-node-(1.14,1.13,1.12,1.11)-v20190927 | v1.5.4
```

Comment:
I don't see 1.15 in here. Should it be?

Reply:
I am not sure - our jenkins tests build 1.15 -

Reply:
this is how it is from upstream, so we'll leave it as is